### ISSUE TITLE
Use releases/tags ?

### ISSUE THREAD

**Issue by JBBianchi** [serverlessworkflow/serverlessworkflow.github.io#215]
I was wondering if we should also use releases/tags on this repository.

Currently, the dashboard of Synapse uses the [specification repo's tags](https://github.com/serverlessworkflow/specification/tags) to define what is the latest version. Then to be able to get the schema as both YAML and JSON, it needs to rely on the website repo. But as there is often a delay between a release on the specification and the website update (which kinda breaks Synapse editor in that interval). 
So I wonder if it would be a good idea to also have releases/tags on this repo ?

**Comment by cdavernas** [comment]
@JBBianchi I'm game! WDYT @ricardozanini?

**Comment by ricardozanini** [comment]
Yes, but maybe a GHA to add tags to all our repos within the org?
